otx/algorithms/action/adapters/mmaction/models/recognizers/recognizer3d.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import functools
+from collections import OrderedDict
+from mmaction.models.builder import RECOGNIZERS
+from mmaction.models.recognizers.recognizer3d import Recognizer3D
+
+
+@RECOGNIZERS.register_module(force=True)
+class Recognizer3D(Recognizer3D):
```

Comment:
if name of child class is same with parent calss, some conflictions or confusion exists later.
how about changing name?

Reply:
Could you explain more about your concern?
And.. the core reason why I didn't keep the class name is to use official mmaction backbone config. For public backbone support, I think this code is convenient for us.